NineChronicles.Standalone.Tests/GraphTypes/StandaloneMutationTest.cs

```diff
@@ -401,6 +327,106 @@ public async Task DailyReward()
+            Assert.Equal(
+                new Dictionary<string, object>
+                {
+                    ["stageTx"] = true,
+                }, 
+                result.Data
+            );
+            Block<PolymorphicAction<ActionBase>> mined =
+                await StandaloneContextFx.BlockChain.MineBlock(service.PrivateKey.ToAddress());
+            Assert.Contains(tx, mined.Transactions);
+        }
+
+        private NineChroniclesNodeService CreateNineChroniclesNodeService(
+            Block<PolymorphicAction<ActionBase>> genesis
+        )
+        {
+            var privateKey = new PrivateKey();
+            var properties = new LibplanetNodeServiceProperties<PolymorphicAction<ActionBase>>
+            {
+                Host = System.Net.IPAddress.Loopback.ToString(),
+                AppProtocolVersion = default,
+                GenesisBlock = genesis,
+                StorePath = null,
+                StoreStatesCacheSize = 2,
+                PrivateKey = privateKey,
+                Port = null,
+                MinimumDifficulty = 4096,
+                NoMiner = true,
+                Render = false,
+                Peers = ImmutableHashSet<Peer>.Empty,
+                TrustedAppProtocolVersionSigners = null,
+            };
+            return new NineChroniclesNodeService(properties, null)
+            {
+                PrivateKey = privateKey
+            };
+        }
+
+        private Block<PolymorphicAction<ActionBase>> MakeGenesisBlock(
+            Address adminAddress,
+            Currency curreny,
+            IImmutableSet<Address> activatedAccounts
+        ) => BlockChain<PolymorphicAction<ActionBase>>.MakeGenesisBlock(
+                new PolymorphicAction<ActionBase>[]
+                {
+                    new InitializeStates(
+                        rankingState: new RankingState(),
+                        shopState: new ShopState(),
+                        gameConfigState: new GameConfigState(_sheets[nameof(GameConfigSheet)]),
+                        redeemCodeState: new RedeemCodeState(Bencodex.Types.Dictionary.Empty
+                            .Add("address", RedeemCodeState.Address.Serialize())
+                            .Add("map", Bencodex.Types.Dictionary.Empty)
+                        ),
+                        adminAddressState: new AdminState(adminAddress, 1500000),
+                        activatedAccountsState: new ActivatedAccountsState(activatedAccounts),
+                        goldCurrencyState: new GoldCurrencyState(curreny),
+                        goldDistributions: new GoldDistribution[0],
+                        tableSheets: new Dictionary<string, string>(),
+                        pendingActivationStates: new PendingActivationState[]{ }
+                    ),
+                }
+            );
+
         private BlockChain<PolymorphicAction<ActionBase>> GetContextFx(PrivateKey playerPrivateKey, RankingState ranking)
```

Comment:
Is there any code left that references this method?

Reply:
There are three refs

- https://github.com/planetarium/NineChronicles.Standalone/blob/50f04d560d5c44d5eb8441a110936c315853c123/NineChronicles.Standalone.Tests/GraphTypes/StandaloneMutationTest.cs#L202
- https://github.com/planetarium/NineChronicles.Standalone/blob/50f04d560d5c44d5eb8441a110936c315853c123/NineChronicles.Standalone.Tests/GraphTypes/StandaloneMutationTest.cs#L235
- https://github.com/planetarium/NineChronicles.Standalone/blob/50f04d560d5c44d5eb8441a110936c315853c123/NineChronicles.Standalone.Tests/GraphTypes/StandaloneMutationTest.cs#L281